src/neo/SmartContract/ApplicationEngine.Runtime.cs

```diff
@@ -228,6 +235,17 @@ protected internal int GetInvocationCounter()
             return counter;
         }
 
+        /// <summary>
+        /// The implementation of System.Runtime.GetRandom.
+        /// Gets the random number genrated form the VRF
+        /// Primary geterates this random number with `prevHash`, the hash of the previous (validators.Length/3 + 1) block
+        /// </summary>
+        /// <returns>The last four bytes of the random number.</returns>
+        protected internal uint GetRandom()
+        {
+            return PersistingBlock.Header.Nonce;
+        }
```

Comment:
What if the contract needs multiple random numbers? Maybe we can use `nonce` as the seed of the random number generator.

Reply:
Added